x-pack/plugins/observability/public/typings/data_handler/index.d.ts

```diff
@@ -25,9 +32,47 @@ interface Series {
 interface FetchDataResponse {
   title: string;
   appLink: string;
-  stats: Stat[];
   series: Series[];
```

Comment:
Would it make sense to change `series` to an explicit key/value object as well like stats? That would give more flexibility in the ui for displaying them.

Reply:
Thinking about this more: maybe we don't event want `stats` and `series`. We'll just want a generic interface like:
```ts
interface FetchDataResponse {
  title: string;
  appLink: string;
  data: {
    metricSeries: Series;
    cpu: Percentage;
    memory: Percentage;
    disk: Percentage;
    inboundTraffic: Bytes;
    outboundTraffic: Bytes;
  }
}
```

Reply:
What I like in the `stats` and `series` is that we have a clear separation of types. Like the metrics `inboundTraffic` and `outboundTraffic` we need have both values in `stats` and series, since we need to show the chart bellow the value:
<img width="400" alt="Screenshot 2020-06-18 at 15 28 29" src="https://user-images.githubusercontent.com/55978943/85026107-90197200-b178-11ea-8317-fc0b67eb2cb7.png">

The Metrics response type would look like:
```js
{
  stats: {
    inboundTraffic: {bytes: 1024},
    outboundTraffic: {bytes: 1024},
  },
  series: [
    {id: 'inboundTraffic', coordinates:[...]},
    {id: 'outboundTraffic', coordinates:[...]},
  ]
}
```

Reply:
But I like the idea of making the series a key/value object.

Reply:
That works too 👍